src/main/java/core/basesyntax/service/impl/CsvReportGenerator.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+import java.util.Map.Entry;
+
+public class CsvReportGenerator implements ReportGenerator {
+    private static final String REPORT_FIRST_LINE = "fruit,quantity";
+    private static final String COMA = ",";
+
+    @Override
+    public String generateReport(Map<String, Integer> storageMap) {

```

Comment:
StorageDao should be injected here via constructor, and from it you could take an actual storage